consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/pki/DefaultKeyStoreWrapperProvider.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
License copyright is now meant to be "Copyright Hyperledger Besu Contributors." https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy